run_daemons.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Start all daemons
+
+nic=$1
```

Comment:
since it is accepting parameters, it would be good to have a -h option that would throw the usage.